api/catalog/settings.py

```diff
@@ -39,7 +39,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DJANGO_DEBUG_ENABLED", default=False, cast=bool)
 
-PYTHON_ENV = config("PYTHON_ENV", default="production")
+ENVIRONMENT = config("ENVIRONMENT", default="local")
```

Comment:
Why did it change to "local"?

Reply:
It's the default that is used by the ingestion_server when it accesses this same variable:

https://github.com/WordPress/openverse-api/blob/ad2f74a111af4ea35f939ec597f394c83f6dbc23/ingestion_server/ingestion_server/distributed_reindex_scheduler.py#L73

Reply:
I think we should keep "production" and change to "staging" or the corresponding string in the staging server. "local" doesn't seem very useful as everything is local to its own machine.

Reply:
Yea I agree that naming convention isn't too useful! :sweat_smile: I think `dev` or `staging` makes sense. A lot of the environment checks in the ingestion server are checking explicitly if in prod (`environment == "prod"`) so it hasn't mattered much what _not prod_ is.

Reply:
I agree `local` is ambiguous.

In my mind (and indeed I've seen this in practice) `dev` is local development, `staging` is the pre-production deployment, `prod` is production, and `test` is when tests are running regardless of environment. Also variations on those strings exist. Node (and most JavaScript projects like React and Vue) uses the full `development` and `production` specifically for example, so I like the consistency of using that for the less-opinionated Python environment.